examples/cargo/BUILD

```diff
@@ -0,0 +1,28 @@
+load(
+    "@io_bazel_rules_rust//rust:rust.bzl",
+    "rust_binary",
+    "rust_library",
+    "rust_test",
+)
+load(
```

Comment:
Any reason for two separate load statements?

Reply:
The path to include is different. Since this is not purely a rust part but a cargo interaction part it lives in a different package.